spec/models/participant_spec.rb

```diff
@@ -20,27 +20,126 @@
+      end
+    end
+    context 'displays participant information' do
+      it 'returns valid string' do
+        expect(participant.display_information(:name)).to eql participant.name
+      end
+    end
+    context 'displays number information' do
+      it 'returns valid string' do
+        expected = '#' + participant.number.to_s
+        expect(participant.display_information(:number)).to eql expected
+      end
+    end
+    context 'displays name' do
+      it 'shows name' do
+        participant.update! name: 'Person'
+        expect(participant.display_information(:name)).to eql 'Person'
+      end
+    end
+    context 'displays three options' do
+      it 'splits up options' do
+        exp = participant.name.to_s + ' (' + participant.bus.number +
+              ', #' + participant.number.to_s + ')'
+        expect(participant.display_information(:name, :bus, :number)).to eql exp
+      end
+    end
+  end
+  describe 'next_number' do
+    it 'returns the last non-nil participant number' do
+      create :participant
+      p_2 = create :participant
+      expect(Participant.next_number).to eql p_2.number + 1
+    end
+  end
 end
+describe 'scoreboard order' do
+  context 'total score' do
+    it 'sorts participants by total score' do
+      per = create :onboard_judging, :perfect
+      per_2 = create :onboard_judging, minutes_elapsed: 8
+      per_3 = create :onboard_judging, minutes_elapsed: 9
+      p_1 = per.participant
+      p_2 = per_2.participant
+      p_3 = per_3.participant
+      expected = [p_1, p_2, p_3]
+      expect(Participant.scoreboard_order(:total_score)).to eql expected
+    end
+  end
+  context 'maneuver score' do 
+    it 'sorts participants by maneuver score' do 
+      per = create :maneuver_participant, :perfect_score
+      per_2 = create :maneuver_participant, :perfect_score, reversed_direction: 1
+      per_3 = create :maneuver_participant, :perfect_score, reversed_direction: 2
+      p_1 = per.participant
+      p_2 = per_2.participant
+      p_3 = per_3.participant
+      expected = [p_1, p_2, p_3]
+      expect(Participant.scoreboard_order(:maneuver_score)).to eql expected
+    end
+  end
+  context 'participant_name' do 
+    it 'sorts participants by participant name' do
+      p_1 = create :participant, name: 'Akiva'
+      p_2 = create :participant, name: 'Arta'
+      p_3 = create :participant, name: 'Molly'
+      expected = [p_1, p_2, p_3]
+      expect(Participant.scoreboard_order(:participant_name)).to eq expected
+    end
+  end
+  context 'participant_number' do
+    it 'sorts participants by participant number' do 
+      p_1 = create :participant
+      p_2 = create :participant
+      p_3 = create :participant
+      expected = [p_1, p_2, p_3]
+      expect(Participant.scoreboard_order(:participant_number)).to eq expected
+    end
+  end
+end
+
+describe 'top_20' do 
+  context'doesnt include last person' do
+    it 'holds 20 people' do
+      21.times { create :participant }
```

Comment:
Seems a little bit fragile - why in particular is it the last participant you create that should be excluded?